### PR TITLE
Facepile - Adding OnRenderPersona and OnRenderPersonaCoin as optional…

### PR DESCRIPTION
**Pull request by makathur**
This is a add on to introduce optional props in facepile component. 

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

1) Introduce optional parameter to override onRenderPersona
2) Introduce optional parameter to override onRenderPersonaCoin

#### Focus areas to test

Facepile

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9480)

### CODE REVIEW

**Comment by KevinTCoughlin** [comment]
Seems reasonable. @Markionium may I request your review?

**Comment by makathur** [comment]
> Just a small comment regarding the persona render function. It seems it is only used when there is just a single person. Perhaps we could add that to the comment of the prop, so that it shows up in the documentation?

**Comment by makathur** [comment]
Resolved the comments and added change , update api as suggested by @micahgodbolt

File: packages/office-ui-fabric-react/src/components/Facepile/Facepile.types.ts [hunk 1]
```diff
@@ -74,6 +74,12 @@ export interface IFacepileProps extends React.ClassAttributes<FacepileBase> {
   /** Type of overflow icon to use */
   overflowButtonType?: OverflowButtonType;
 
+  /** Optional custom renderer for the persona , gets called only when their is one persona in personas array*/
```
Comment (Markionium) [inline packages/office-ui-fabric-react/src/components/Facepile/Facepile.types.ts]:
```suggestion
  /** Optional custom renderer for the persona, gets called only when there is one persona in the personas array*/
```

File: packages/office-ui-fabric-react/src/components/Facepile/Facepile.types.ts [hunk 2]
```diff
@@ -74,6 +74,12 @@ export interface IFacepileProps extends React.ClassAttributes<FacepileBase> {
...
+  onRenderPersona?: IRenderFunction<IFacepilePersona>;
+
+  /** Optional custom renderer for the persona coins, gets called only when their is more than one persona in personas array*/
```
Comment (Markionium) [inline packages/office-ui-fabric-react/src/components/Facepile/Facepile.types.ts]:
```suggestion
  /** Optional custom renderer for the persona coins, gets called only when there is more than one persona in the personas array*/
```

**Comment by micahgodbolt** [comment]
quick tweaks to changefile and docs. will set automerge to push this in once CI is done.

**Pull request merged**